lib/certs.js

```diff
@@ -0,0 +1,95 @@
+/*jslint node: true */
+'use strict';
+
+const os = require('os');
+const path = require('path');
+const fs = require('fs-extra');
+const logger = require('@blackbaud/skyux-logger');
+const npmInstall = require('./npm-install');
+const cloneUtil = require('./utils/clone');
+
+function getCertRootPath() {
+  return path.resolve(`${os.homedir()}/.skyux/certs/`);
+}
+
+function getResolver() {
+  return require(getCertRootPath());
+}
+
+async function clean() {
+  const certRootPath = getCertRootPath();
+
+  if (fs.existsSync(certRootPath)) {
+    logger.info(`Removing previous clone of certs from ${certRootPath}`);
+    await fs.remove(certRootPath);
+  } else {
+    logger.info(`Previous clone of certs not found at ${certRootPath}`);
+  }
+}
+
+async function clone() {
+  const certRootPath = getCertRootPath();
+  await cloneUtil('https://github.com/blackbaud/skyux-sdk-certs.git', certRootPath, argv);
+  await npmInstall({ path: certRootPath });
```

Comment:
This line jumped out at me, but I'm not sure what it's doing?

Reply:
Just confirming, you're talking about the `npmInstall` line?  That's simply running `npm install` in the context of the cloned `skyux-sdk-certs` directory.  Same technique and method we use in the `skyux new` functionality.